docs/templating-language.md

```diff
@@ -1810,13 +1814,14 @@ This can also be used with a pipe function.
 ## Nomad Functions
 
 Nomad service registrations can be queried using the `nomadServices` and `nomadService` functions.
+Nomad variables can be queried using the `nomadVarList` and `nomadVar` functions.
```

Comment:
It might be nice to link Nomad variables to https://www.nomadproject.io/docs/concepts/variables (and Nomad service registrations to https://www.nomadproject.io/docs/job-specification/service#provider )